core/utils/poly-look/src/react/tabs/tabs.css

```diff
@@ -1,21 +1,25 @@
-.tab {
+.tabs .tab-buttons {
     margin: 0 auto;
     display: flex;
+    flex-direction: row;
```

Comment:
`flex-direction: row` is the default, are you adding it for any particular reason? (maybe in case the parent is `flex-direction: column`?) If not, I think we could safely delete it